dask/array/backends.py

```diff
@@ -160,11 +169,18 @@ def _concat_cupy_sparse(L, axis=0):
 
 @tensordot_lookup.register_lazy("sparse")
 @concatenate_lookup.register_lazy("sparse")
+@nannumel_lookup.register_lazy("sparse")
+@numel_lookup.register_lazy("sparse")
 def register_sparse():
     import sparse
 
     concatenate_lookup.register(sparse.COO, sparse.concatenate)
     tensordot_lookup.register(sparse.COO, sparse.tensordot)
+    # Enforce ndarray for the numel result, since the sparse
```

Comment:
I think this is a correct statement (that seems a bit more clear to me)

```suggestion
    # Enforce dense ndarray for the numel result, since the sparse
```